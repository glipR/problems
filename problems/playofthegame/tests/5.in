2 20
a
d|s|d|wad|dsw|ws||
s wda|wsd|sw||
