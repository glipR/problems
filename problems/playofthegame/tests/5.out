Player #2 matched 5 frames starting at frame #4.
